.github/workflows/push.yml

```diff
@@ -88,3 +99,41 @@ jobs:
             -d opcache.jit_buffer_size=16M
       - name: Verify generated files are up to date
         uses: ./.github/actions/verify-generated-files
+  WINDOWS:
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - x64: true
+            zts: false
+          - x64: false
+            zts: true
+    name: "WINDOWS_${{ matrix.x64 && 'X64' || 'X86' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}"
+    runs-on: windows-2019
+    env:
+      APPVEYOR_REPO_BRANCH: "${{ github.ref_name }}"
+      PHP_BUILD_CACHE_BASE_DIR: c:\build-cache
+      PHP_BUILD_OBJ_DIR: c:\obj
+      PHP_BUILD_CACHE_SDK_DIR: c:\build-cache\sdk
+      PHP_BUILD_SDK_BRANCH: php-sdk-2.2.0
+      PHP_BUILD_CRT: vs16
+      PLATFORM: ${{ matrix.x64 && 'x64' || 'x86' }}
+      THREAD_SAFE: "${{ matrix.zts && '1' || '' }}"
+      INTRINSICS: "${{ matrix.zts && 'AVX' || '' }}"
+    steps:
+      - name: git checkout 1/2
+        run: git config --global core.autocrlf false && git config --global core.eol lf
+      - name: git checkout 2/2
+        uses: actions/checkout@v2
+      - name: Setup MySQL
+        run: |
+          choco install mysql -y --params "/port:3307"
+          mysql --port=3307 --user=root --password="" -e "ALTER USER 'root'@'localhost' IDENTIFIED BY 'mysql_pass!'; FLUSH PRIVILEGES;"
+      - name: Setup MSSQL
+        run: |
+          choco install sql-server-express -y
+          sqlcmd -S "$env:COMPUTERNAME/SQLEXPRESS" -Q "select 1;"
```

Comment:
I tried various server names here incl. tcp:ip,port, but all options fails to connect.

Can anyone advise what option to use with `-S`?

Reply:
Did you try without the `-S` option? According to the [docs](https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility?view=sql-server-ver15):

> If you do not specify a server_name [ \instance_name ] when you start sqlcmd, SQL Server checks for and uses the SQLCMDSERVER environment variable.

Reply:
yes, I tried, and the `SQLEXPRESS` namespace is correct there as it is defined in the choco install code